lab_4_auto_completion/main.py

```diff
@@ -263,6 +493,47 @@ def build(self) -> int:
         Returns:
             int: 0 if attribute is filled successfully, otherwise 1
         """
+        try:
```

Comment:
reduce amount of code in `try` block, you have to put only dangerous code in this block